code/modules/surgery/tools.dm

```diff
@@ -77,6 +77,13 @@
 	tool_behaviour = TOOL_DRILL
 	toolspeed = 1
 
+/obj/item/surgicaldrill/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] rams [src] into [user.p_their()] chest! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	addtimer(CALLBACK(user, /mob/living/carbon.proc/gib, null, null, TRUE, TRUE), 25)

```

Comment:
```suggestion
	addtimer(CALLBACK(user, /mob/.proc/gib, null, null, TRUE, TRUE), 25)
```

Reply:
Compile time error since apparently some subtypes of /mob override too many args, and all of the mobs (such as the AI) can't suicide with a drill anyways.

Reply:
lamee and g*y